### PR TITLE
Create the Judge Strike team

### PR DESCRIPTION
**Pull request by Manleto**
Are you sick and tired of the fact that nine times out of 10 the only person to judge a court case is a biased captain? Well now (and this probably won't work) you can pick some poor shlub out of the void to judge for you!. 

:cl:
 * rscadd: Added a judge strike team for obs

### CODE REVIEW

**Comment by gurfan** [comment]
does this work? did you test this?

**Comment by Manleto** [comment]
> does this work? did you test this?

Not in the slightest, I have no idea if this works.

File: code/game/striketeams/Judge [hunk 1]
```diff
@@ -0,0 +1,162 @@
+//Judge
+
+var/list/response_team_members = list()
+
+/datum/striketeam/Judge
+	striketeam_name = Judge
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:6]: this ain't a DEFINE chief

File: code/game/striketeams/Judge [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	mission = "preside over this case and come up with an equitable decision."
+	team_size = 1
+	min_size_for_leader = 0//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	spawns_name = "Judge"
+	can_customize = TRUE
+	logo = "Judge-logo"
+	outfit_datum = /datum/outfit/striketeam/judge
+
+/datum/striketeam/Judge/failure()
+	command_alert(/datum/command_alert/judge_fail)
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:17]: I do not believe you have added a command alert

File: code/game/striketeams/Judge [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+/datum/striketeam/Judge/extras()
+	command_alert(/datum/command_alert/ert_success)
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:20]: just the default ert message?

File: code/game/striketeams/Judge [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+/datum/striketeam/Judge/greet_commando(var/mob/living/carbon/human/H)
+	if(H.key == leader_key)
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:23]: given that there's a team size of 1 and min size for leader 0, this should always be true?

File: code/game/striketeams/Judge [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+	if(spawner)
+		spawner.occupant = null
+		spawner.update_icon()
+
+	M.forceMove(spawn_location.loc)
+
+	return M
+
+//stealing that sweet bobbing animation from cryo.dm
+/obj/machinery/ert_cryo_cell
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:95]: why would you copy paste this object? it already exists in the other file?

File: code/game/striketeams/Judge [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+/datum/striketeam/Judge/greet_commando(var/mob/living/carbon/human/H)
+	if(H.key == leader_key)
+		to_chat(H, "<span class='danger'>You are [H.real_name], the Judge. You answer only to Central Command and are expected to follow Space Law. Assist your assigned station to the best of your abilities.</span>")
+	else
+		to_chat(H, "<span class='danger'>You are [H.real_name], Paralegal. You answer only to Central Command and are expected to follow Space Law. Assist your assigned station to the best of your abilities.</span>")
+	for (var/role in H.mind.antag_roles)
+		var/datum/role/R = H.mind.antag_roles[role]
+		R.AnnounceObjectives()
+
+/datum/striketeam/Judge/create_commando(obj/spawn_location, leader_selected = 0, key = "")
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:31]: you're completely recycling the ERT spawn here, meaning that judges will come to the station after fully walking through the candyshop of ERT gear... delay trial by another 10 minutes as he chooses his favorite selection of guns, battering ram, tools, autoinjectors, and SCP

File: code/game/striketeams/Judge [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+/datum/striketeam/Judge
+	striketeam_name = Judge
+	faction_name = "Nanotrasen"
+	mission = "preside over this case and come up with an equitable decision."
+	team_size = 1
+	min_size_for_leader = 0//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	spawns_name = "Judge"
+	can_customize = TRUE
+	logo = "Judge-logo"
+	outfit_datum = /datum/outfit/striketeam/judge
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:14]: does this outfit datum exist?

File: code/game/striketeams/Judge [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+
+/datum/striketeam/Judge
+	striketeam_name = Judge
+	faction_name = "Nanotrasen"
+	mission = "preside over this case and come up with an equitable decision."
+	team_size = 1
+	min_size_for_leader = 0//set to 0 so there's always a designated team leader or to -1 so there is no leader.
+	spawns_name = "Judge"
+	can_customize = TRUE
+	logo = "Judge-logo"
```
Comment (Kurfursten) [inline code/game/striketeams/Judge:13]: This logo does not exist

File: code/game/striketeams/Judge (hunk 6 above)
Comment (Manleto) [reply code/game/striketeams/Judge:31]: Yes

File: code/game/striketeams/Judge (hunk 7 above)
Comment (Manleto) [reply code/game/striketeams/Judge:14]: It does not

File: code/game/striketeams/Judge (hunk 8 above)
Comment (Manleto) [reply code/game/striketeams/Judge:13]: yes

**Comment by gurfan** [comment]
epic

**Comment by Eneocho** [comment]
ebin

**Pull request closed without merging**